config/locales/models/motif.fr.yml

```
@@ -4,13 +4,16 @@ fr:
       motif: Motif
     attributes:
       motif:
+        name: Nom
         organisation: Organisation
         service: Service
         collectif: Nombre de participants
         color: Couleur
         bookable_by: Réservation en ligne
-        bookable_by_hint: Vous pouvez décider de qui peut prendre RDV pour ce motif
+        bookable_by_hint: "Définissez quel utilisateur peut prendre rendez-vous pour ce motif :"
         visibility_type: Visibilité usager
+        booking_delay: Délai de réservation
+        booking_delay_hint: "Vous pouvez définir une “fenêtre de réservation” pour la prise d’un rendez-vous : il s’agit du délai maximum et minimum possible, entre le moment où l’usager prend rendez-vous, et le rendez-vous en lui même."
```

👍 c'est cool l'usage du terme "fenêtre de réservation", c'est plus explicite

---

On pourrait clarifier que cette fenêtre ne s'applique pas aux rdv pris par les agents ?
```suggestion
        booking_delay_hint: "Vous pouvez définir une “fenêtre de réservation” pour la prise d’un rendez-vous : il s’agit du délai maximum et minimum possible, entre le moment où l’usager prend rendez-vous, et le rendez-vous en lui même. Ces délais ne s'appliquent pas à la prise de rendez-vous par d'autres agents."
```